scc/delivery-with-carvel-app.hbs.md

```diff
@@ -0,0 +1,210 @@
+
+## Create App
+
+1. You need to give the Build cluster access to the Run clusters. On the Build cluster, for each Run cluster, create a `Secret` containing the Run cluster's kubeconfig:
+
+   ```console
+   kubectl create secret generic <run-cluster>-kubeconfig \
+       -n <build-cluster-ns> \
+       --from-file=value.yaml=<path-to-run-cluster-kubeconfig>
+   ```
+
+2. The Carvel `App` custom resource represents a collection of Kubernetes resources that kapp-controller can fetch and deploy to a cluster. The `App` will be pointed at the git repository branch where kapp-controller resources (e.g. `PackageRepository` and `Packages`) will be defined. By default, an `App` custom resource will sync the cluster with its fetch source every 30 seconds to prevent the cluster state from drifting from its source of truth, which is a git repository in this case. Create the following `App` on your Build cluster:
+
+   ```yaml
+   ---
+   apiVersion: kappctrl.k14s.io/v1alpha1
+   kind: App
+   metadata:
+     name: hello-app-app
+     namespace: <build-cluster-ns>
+   spec:
+     # specifies that app should be deployed authenticated via the
+     # given service account, found in this namespace
+     serviceAccountName: <SERVICE_ACCOUNT>
+     # specifies that app should be deployed to destination cluster;
+     # by default, cluster is same as where this resource resides
+     cluster:
+       # specifies namespace in destination cluster
+       namespace: ns2
+       # specifies secret containing kubeconfig
+       kubeconfigSecretRef:
+         # specifies secret name within app's namespace
+         name: cluster1
+         # specifies key that contains kubeconfig
+         key: value
+     fetch:
+     - git:
+         url: # GitOps repo URL ex: https://github.com/mycompany/my-gitops
+         ref: # GitOps repo branchex: origin/main
+         subPath: <path_for_packages> # ex: hello-app.dev.tap/packages/
+     - git:
+         url: # GitOps repo URL ex: https://github.com/mycompany/my-gitops
+         ref: # GitOps repo branch ex: origin/main
+         subPath: <path_for_package_installs> # ex: hello-app.dev.tap/runcluster1
+     template:
+     - ytt: {}
+
+      deploy:
+     - kapp:
+         intoNs: <DESIRED_NAMESPACE>
+         rawOptions: ["--dangerous-allow-empty-list-of-resources=true"]
+   ```
+
+   > **Note:** The fetch section can includes entries for all the locations in the gitops repo to be deployed, and be appended with other run clusters if needed. Each App CR must specify either a service account (via `spec.serviceAccountName`) or, a `Secret` with kubeconfig contents for some Run cluster (via `spec.cluster.kubeconfigSecretRef.name`).
+
+3. You will also need to create a `ServiceAccount` with associated RBAC permissions for the App to use. Example of the required permissions for the `ServiceAccount`. Make sure to assess appropriate RBAC needed for your specific PackageInstalls.
```

Comment:
I think we need more clarification on service accounts. We have a section up above where we create a service account on a run cluster, and then this section for the build cluster. I don't think it's clear where the user should be using both of those service accounts. Also, this section should be before we create the `App` because it's a prereq.